<!DOCTYPE html>
<html><head><title>GetDatasourceCredential</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">5 references to GetDatasourceCredential</div><div class="rA">Azure.AI.MetricsAdvisor (1)</div><div class="rG" id="Azure.AI.MetricsAdvisor"><div class="rF"><div class="rN">MetricsAdvisorAdministrationClient.cs (1)</div>
<a href="../MetricsAdvisorAdministrationClient.cs.html#1955"><b>1955</b>var createdCredential = <i>GetDatasourceCredential</i>(credentialId, cancellationToken);</a>
</div>
</div>
<div class="rA">Azure.AI.MetricsAdvisor.Tests (4)</div><div class="rG" id="Azure.AI.MetricsAdvisor.Tests"><div class="rF"><div class="rN">MetricsAdvisorAdministrationClient\DatasourceCredentialTests.cs (4)</div>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DatasourceCredentialTests.cs.html#82"><b>82</b>Assert.That(() =&gt; adminClient.<i>GetDatasourceCredential</i>(null), Throws.InstanceOf&lt;ArgumentNullException&gt;());</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DatasourceCredentialTests.cs.html#83"><b>83</b>Assert.That(() =&gt; adminClient.<i>GetDatasourceCredential</i>(&quot;&quot;), Throws.InstanceOf&lt;ArgumentException&gt;());</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DatasourceCredentialTests.cs.html#84"><b>84</b>Assert.That(() =&gt; adminClient.<i>GetDatasourceCredential</i>(&quot;credentialId&quot;), Throws.InstanceOf&lt;ArgumentException&gt;().With.InnerException.TypeOf(typeof(FormatException)));</a>
<a href="../../Azure.AI.MetricsAdvisor.Tests/MetricsAdvisorAdministrationClient/DatasourceCredentialTests.cs.html#96"><b>96</b>Assert.That(() =&gt; adminClient.<i>GetDatasourceCredential</i>(FakeGuid, cancellationSource.Token), Throws.InstanceOf&lt;OperationCanceledException&gt;());</a>
</div>
</div>
</body></html>